chart/templates/gateway/configmap.yaml

```diff
@@ -35,7 +35,6 @@ data:
       "^GET /api/infra/\\d+/auto_fixes(?:\\?|$|/$)",
       "^GET /api/infra/\\d+/delimited_area(?:\\?|$|/$)",
       "^GET /api/infra/\\d+/routes/track_ranges(?:\\?|$|/$)",
-      "^GET /infra/\\d+/lines/\\d+/bbox(?:\\?|$|/$)",
```

Comment:
The is another occurrence of this in the `routin_exclude` list.

Could you keep this deletion in a separate commit?